content/docs/08-user-management/4-saml-sso.md

```diff
@@ -6,99 +6,85 @@ icon: ""
+   2.  **OpenID Connect (OIDC) Based SSO** - You can use OIDC to enable OIDC SSO between Palette and the recommended IdP. It requires application registration to issue a Client ID, Client Secret, and Validation.<p></p><br />
 
 <br />
 
+----
 
 <Tabs>
 <Tabs.TabPane tab="SAML BASED SSO" key="saml">
 
-# How to Set Up SAML 2.0-Based SSO within Palette
+# How to Set Up SAML 2.0-based SSO within Palette
 
-With Palette, you can use SAML 2.0 single sign-on (SSO) to enable authentication using an IdP when logging into Palette.
+With Palette, you can use SAML 2.0 single sign-on (SSO) to enable the secure exchange of user authentication via an IdP and Spectro Cloud Palette.
 
-## Procedure
+<br />
 
-1. To set up an Identity provider (IdP) based SSO, log in to the Palette console as the Tenant Admin.<p></p><br />
-2. Select the **Tenant Settings** > **SSO Auth Type** > **SAML** to view the SAML panel.<p></p><br />
-3. Complete the assertion form with the requested parameters. See below for more details specific to supported IdPs.<p></p><br />
+## Procedure
 
-     - Issuer - The unique identifier. In this case, it is Palette.
-     - Certificate
-     - Service
-     - Identity Provider Metadata
-     - Default Teams
-         - Enable Single Logout
+To set up Spectro Cloud Palette with an identity provider (IdP) SAML based SSO:<p></p><br />
+1. Log in to the Palette console as a Tenant Admin.<p></p><br />
+2. Select **Tenant Settings** > **SSO Auth Type** > **SAML** to view the SAML panel.<p></p><br />
+3. Complete the assertion form with the requested parameters. See below for more details specific to the supported IdPs.<p></p><br />
+    The following options will be available for configuring SAML SSO within Palette: <p></p><br />
 
-<p></p>
+      - **Service** - Choose your IdP (Azure Active Directory, Okta, Keycloak, OneLogin, ADFS, Other).<p></p><br />
+      - **Identity Provider Metadata** - Enter the Identity Provider Metadata.<p></p><br />
+      - **Default Teams** - Add the authenticated user's Default Team(s) Palette membership.<p></p><br />
+      - **NameID Format** - Choose the appropriate version of the format in use (SAML 1.1, SAML 2.0, email address, other).<p></p><br />
 
-4. These parameters enable Palette as a **Service Provider** (SP) into the IdP:<br />
-   - Single Logout URL
-   - EntityId
-   - NameId Format
-   - Login URL
-   - FirstName
-   - LastName
-   - Email
-   - SpectroTeam
-   - Service Provider Metadata
+    The following parameters will enable Spectro Cloud Palette as a **Service Provider** (SP) in your IdP. Your IdP will require some or all the information listed below to enable SSO with Palette. <p></p><br />
 
-<p></p>
+      - **Single Logout URL** - This is the logout URL. Include the URL into the IdP for the logout to clear the provider-side session. <p></p><br />
```

Comment:
Change to "- **Single Logout URL** - The IdP will need the logout URL for the SSO configuration of SAML. <p></p><br />